desk.css

```diff
@@ -711,11 +712,237 @@ footer .actbutton2 {
   box-sizing: border-box;
 }
 
-/* desk-version */
+/*  popup-window mobile */
+
+.popup-window {
```

Comment:
- [ ] As you can see in the screenshot below your popup modal is under your section elements, and this problem because you didn't specify a `z-index` to the **pop-window** element, Kindly add a big value to your popup `z-index` to be always at the top of your screen

![z-index](https://user-images.githubusercontent.com/80895497/163412384-208d2827-014f-48cd-b521-2308e3b4c351.PNG)